runtime/parachains/src/hrmp.rs

```diff
@@ -339,27 +343,27 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
-	/// Maintains a mapping that can be used to answer the question:
-	/// What paras sent a message at the given block number for a given receiver.
-	/// Invariants:
+	/// Maintains a mapping that can be used to answer the question: What paras sent a message at
+	/// the given block number for a given receiver. Invariants:
 	/// - The inner `Vec<ParaId>` is never empty.
 	/// - The inner `Vec<ParaId>` cannot store two same `ParaId`.
-	/// - The outer vector is sorted ascending by block number and cannot store two items with the same
-	///   block number.
+	/// - The outer vector is sorted ascending by block number and cannot store two items with the
+	///   same block number.
 	#[pallet::storage]
 	pub type HrmpChannelDigests<T: Config> =
 		StorageMap<_, Twox64Concat, ParaId, Vec<(T::BlockNumber, Vec<ParaId>)>, ValueQuery>;
 
 	/// Preopen the given HRMP channels.
 	///
-	/// The values in the tuple corresponds to `(sender, recipient, max_capacity, max_message_size)`,
-	/// i.e. similar to `init_open_channel`. In fact, the initialization is performed as if
-	/// the `init_open_channel` and `accept_open_channel` were called with the respective parameters
-	/// and the session change take place.
+	/// The values in the tuple corresponds to `(sender, recipient, max_capacity,
+	/// max_message_size)`, i.e. similar to `init_open_channel`. In fact, the initialization is
```

Comment:
IMO, the prior version that does not wrap the tuple is more legible since it avoids scanning the line.